src/transformers/models/t5/modeling_t5.py

```diff
@@ -616,7 +666,7 @@ def forward(
 class T5LayerCrossAttention(nn.Module):
     def __init__(self, config):
         super().__init__()
-        self.EncDecAttention = T5Attention(config, has_relative_attention_bias=False)
+        self.EncDecAttention = T5Attention(config, relative_position_embedding_definitions=None)
```

Comment:
So you didn't play with this attention mechanism?

Reply:
I extended T5Attention to support various relative position encodings  either original, None, New (e.g. with position ids), or any combination of  original and new.

Reply:
In this cases = Note that it always used has_relative_attention_bias=False, so I kept the same performance  - did not add the relative position embedding when it did not exist before. We can discuss this